website/content/docs/lambda/registration.mdx

```diff
@@ -0,0 +1,271 @@
+
+
+### Architecture
+
+This diagram shows the flow of events from EventBridge into Consul:
+
+<ImageConfig width={500}>
+
+![Lambda Registrator Architecture](/img/lambda_registrator_architecture.svg)
+
+</ImageConfig>
+
+1. EventBridge invokes Lambda registrator. This is either done based on
+   CloudTrail Lambda events or a schedule.
+2. Lambda registrator determines how to reconcile Lambda's control plane state
+   with Consul state and ensures they are in sync by registering, updating and
+   deregistering Lambda services.
+
+### Deploy Lambda Registrator
+
+#### Deploy Lambda Registrator
+
+Here is an example usage of the Lambda registrator terraform module:
+```hcl
+module "lambda-registrator" {
+  source  = "hashicorp/consul-lambda-registrator/aws//modules/lambda-registrator"
+  name = "consul-lambda-registrator"
+  consul_http_addr = "https://consul.example.com:8501"
+  ca_cert_path = aws_ssm_parameter.ca-cert.name
+  http_token_path = aws_ssm_parameter.acl-token.name
+}
+```
+
+Deploy Lambda registrator with `terraform apply`.
+
+#### Optional: Store the CA Certificate in Parameter Store
+
+When Consul making request to the [HTTP API](/api-docs) over HTTPS, Lambda
+registrator uses a CA Certificate stored in Parameter Store. Consul’s Server CA
+can be stored into Parameter Store by applying the following Terraform:
+
+```hcl
+resource "aws_ssm_parameter" "ca-cert" {
+  name        = "/lambda-registrator/ca-cert"
+  type        = "SecureString"
+  value       = <VALUE>
+}
+```
+
+#### Optional: Store the ACL Token in Parameter Store
+
+When Consul has ACLs enabled, Lambda registrator uses an ACL token stored in
+Parameter Store. This can be done with the CLI, API, or Terraform provider.
+With the CLI, it is done by:
+1. Write the following to rules.hcl:
+```hcl
+service_prefix "" {
+  policy = "write"
+}
+```
+
+2. Create the policy with `consul acl policy create -name "lambda-registrator-policy" -rules @rules.hcl`.
+3. Create the token with `consul acl token create -policy-name "lambda-registrator-policy"`.
+4. Store the token in Parameter Store by applying the following Terraform:
+```hcl
+resource "aws_ssm_parameter" "acl-token" {
+  name        = "/lambda-registrator/acl-token"
+  type        = "SecureString"
+  value       = <VALUE>
+}
+```
+
+#### Configuration Options
+
+| Name | Type |  Description |
+| - | - | - | - |
+| `name` | Configuring AWS services | This is used to name Lambda registrator’s Lambda function and to construct the Identity and Access Management (IAM) role and policy names used by the Lambda function. |
+| `schedule_frequency_in_minutes` | Configuring AWS services | The interval EventBridge uses to trigger a full synchronization. This defaults to 5 minutes. |
+| `timeout` |Configuring AWS services | The maximum number of seconds Lambda registrator can run per invocation before timing out. |
+| `consul_http_addr` | Runtime | This is directly used to configure the Consul API client. |
```

Comment:
```suggestion
| `consul_http_addr` | Runtime | Specifies the address of the Consul API client. |
```
I'm not actually clear what this does. Does it specify the bind address for Consul API clients?

Reply:
Yeah, you are right. This is better.